.github/workflows/dependabot.yml

```diff
@@ -0,0 +1,21 @@
+name: dependabot
+on: pull_request
+
+permissions:
+  pull-requests: write
```

Comment:
I'm not sure this statement actually does anything. If I understand correctly, because this runs on `pull_request` there are only read-only permissions. Did you test this?